src/components/Books.js

```diff
@@ -12,23 +11,18 @@ const Container = styled.div`
 `;
 
 function Books() {
-  const books = useSelector(selectBooks);
+  const books = useSelector((state) => state.books);
   return (
     <Container>
-      {books.length ? (
-        books.map((book) => (
-          <CardBook
-            key={book.id}
-            id={book.id}
-            title={book.title}
-            author="Shedrack Sunday"
-            category={book.category}
-          />
-        ))
-      ) : (
-        <h1>Add new Book!</h1>
-      )}
-
+      {books.map((book) => (
+        <CardBook
+          key={book.item_id}
+          id={book.item_id}
+          name={book.title}
+          author="Shedrack Sunday"
+          category={books.category}
```

Comment:
- [ ] as requested by the previous reviewer, please update your styles to match the design given. Look at the comparison below 👇 
**Your design**
![](https://user-images.githubusercontent.com/83048208/201466689-e092fa5c-250e-4b91-9940-e6cd9625c94d.png)
**Given design**
![](https://user-images.githubusercontent.com/83048208/201466702-b4ec3007-9cb3-48da-866f-57bb52249549.png)

HINT:
- Please rotate the progress bar to match the one on the design 😉

Reply:
This is supposed to be dynamic. Why must it have to be exact? The requirement does not state so. @BertrandConxy

Reply:
This is it from my machine working perfectly. please update all you dependencies. the error is from your end.
![yt](https://user-images.githubusercontent.com/50990925/201470287-a31179ec-2a0d-4603-b452-8cea48cf9de7.png)